scikits/odes/sundials/cvode.pyx

```diff
@@ -752,12 +781,6 @@ cdef class CVODE:
                                'verbosity']:
                     raise ValueError("Option '%s' can''t be set runtime." % opt)
```

Comment:
I also stumbled upon 'verbosity' being totally useless right now. Though I did not remove it as it *may* be useful, and it is harmless. Anyway, if we decide not to support it, the user should be at least warned about its deprecation. But I don't feel this decision belongs to this commit.